pipeline.py

```diff
@@ -1,13 +1,26 @@
 """
 Pipeline for text processing implementation
 """
-
+import requests
```

Comment:
change the name of PR to use Work instead of work - this should make your PR name check green